token/program-2022/src/extension/confidential_transfer/mod.rs

```diff
@@ -110,8 +110,11 @@ pub struct ConfidentialTransferAccount {
     /// The decryptable available balance
     pub decryptable_available_balance: DecryptableBalance,
 
-    /// `pending_balance` may only be credited by `Deposit` or `Transfer` instructions if `true`
-    pub allow_balance_credits: PodBool,
+    /// If `false`, the extended account rejects any incoming confidential transfers
+    pub allow_confidential_credits: PodBool,
+
+    /// If `false`, the base account rejects any incoming confidential transfers
```

Comment:
```suggestion
    /// If `false`, the base account rejects any incoming transfers
```

Reply:
Thanks for the catch! It seems like specifying that "non-confidential" transfers are rejected may prevent potential confusion.